modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,392 @@
+<?php declare(strict_types=1);
+/**
```

Comment:
This part can be deleted. We used to have an automatic test rule to make file comments in PHP code mandatory but it's not anymore.